### DISCUSSION TITLE
Problem with installing Manim in Mac Os

### DISCUSSION THREAD

**Discussion by Arindam1050** [ManimCommunity/manim#3814] [Q&A]
I tried to install the Manim in Mac. I followed the instructions in the 3b1b Manim repository. While I gave the command
>pip install -e .

Some error messages occurred.

>ERROR: pip's dependency resolver does not currently take into account all the packages that are installed. This behaviour is the source of the following dependency conflicts.
spyder 4.1.5 requires pyqt5<5.13; python_version >= "3", which is not installed.
spyder 4.1.5 requires pyqtwebengine<5.13; python_version >= "3", which is not installed.

At last when I tried to use the command 
>manimgl example_scenes.py OpeningManimExample

It is not opening the any example manim page for me. Can you please help me with what to do to resolve the issue? Thank you in advance.

**Comment by uwezi** [comment]
If you have problems installing 3B1B's version of Manim you should post in the appropriate Github repository. This repository is for ManimCE where you will find the installation instructions for MacOS here:

https://docs.manim.community/en/stable/installation/macos.html

https://youtu.be/8DCZPrcCttw?si=ChxxebKduK-FgDOn 

see also:
https://docs.manim.community/en/stable/faq/installation.html#why-are-there-different-versions-of-manim

**Reply by Arindam1050** [reply]
@uwezi Thank you. Sorry to post it here.

**Reply by uwezi** [reply]
Don't be sorry - I just wanted to point out that you cannot expect any help for ManimGL here. And if you want to try ManimCE instead you are welcome over to Discord: https://docs.manim.community/en/stable/faq/general.html?highlight=discord#where-can-i-find-more-resources-for-learning-manim

**Reply by Arindam1050** [reply]
I do not know about ManimCE, I have learnt a little about Manim but I am going to check it. Thank you very much.
